client/src/components/DataEntry/index.test.tsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react'
 import { render, waitFor, fireEvent } from '@testing-library/react'
 import { StaticRouter } from 'react-router-dom'
-import { routerTestProps, asyncActRender } from '../testUtilities'
+import { routerTestProps } from '../testUtilities'
 import DataEntry from './index'
 import { dummyBoards, dummyBallots, contest } from './_mocks'
 import * as utilities from '../utilities'
 
+window.scrollTo = jest.fn()
```

Comment:
@jonahkagan Not blocking, just curious what this does.

Reply:
There was a warning about `window.scrollTo` being undefined (we use `window.scrollTo` to reset to the top of the screen when you transition between ballots). This is what the internet said would fix it, and it did :)